apps/server/src/modules/provisioning/service/tsp-provisioning.service.spec.ts

```diff
@@ -204,6 +216,7 @@ describe('TspProvisioningService', () => {
 				const classId = faker.string.uuid();
 				const externalClass = externalClassDtoFactory.build({
 					externalId: classId,
+					name: withName ? faker.string.uuid() : '',

```

Comment:
I would consider using https://fakerjs.dev/api/person.html
```suggestion
					name: withName ? faker.person.fullName() : '',
```